### ISSUE TITLE
Centrifugo 1.6.x can close connection without any reason

### ISSUE THREAD

**Issue by johndoejdg** [centrifugal/centrifugo#142]
Centrifugo 1.6.1 run locally on 127.0.0.1 with memory engine.
After some random actions on page it can suddenly close connection though I do not leave page or close connection by javacript API.

Server logs:
![image](https://cloud.githubusercontent.com/assets/2064465/22057974/36d0f15a-dd78-11e6-88da-725d132f38e8.png)

Client logs: 
![image](https://cloud.githubusercontent.com/assets/2064465/22057996/5029f2a0-dd78-11e6-9d1a-6ce782d12990.png)

Server config:
```
{
  "secret": "sec"
  "admin_password": "pass",
  "admin_secret": "admin_sec",
  "join_leave": false,
  "anonymous": false,
  "publish": true,
  "watch": true,
  "presence": true,
  "history_size": 100,
  "history_lifetime": 1800,
  "recover": true,
  "history_drop_inactive": true,
  "namespaces": [
    {
      "name": "system",
      "publish": true,
      "anonymous": true,
      "watch": true,
      "presence": true,
      "join_leave": true,
      "history_size": 0,
      "history_lifetime": 0,
      "history_drop_inactive": false
    }
  ]
}
```
Last messages before disconnect:
![image](https://cloud.githubusercontent.com/assets/2064465/22058502/20c8974e-dd7a-11e6-9889-bd9a0c9618ab.png)

Centrifugo web admin "messages" section does not show any big messages. And can not be problem with network because it run locally.

**Comment by johndoejdg** [comment]
Same for 1.6.2

**Comment by FZambia** [comment]
Thanks for a report!

Do you use latest javascript client? Could you try to reproduce this when `ping: false` in js client options?

**Comment by johndoejdg** [comment]
Yes, `ping: false` solve problem.

On client side centrifugo-js: 1.4.1 without sockjs.

**Comment by FZambia** [comment]
Yeah, I see a problem - it's introduced with 1.6.0, will fix it very soon.

**Comment by FZambia** [comment]
@johndoejdg fixed in 1.6.3

Also I suggest updating `centrifuge-js` to version 1.4.2 for those who use SockJS polling transports.
